content/en/_index.md

```diff
@@ -1068,5 +1320,9 @@ end
 
 [minitest]: https://github.com/minitest/minitest
 
+{{< /content-tab >}}
+{{< content-tab header="Terraform" >}}
+For information about testing, see the Terraform samples [contributing guide]
```

Comment:
Thow into comment block. markdown hyperlink does not render in tabs

Reply:
removed